### DISCUSSION TITLE
Automazione di 4 interruttori wifi

### DISCUSSION THREAD

**Discussion by dockydock** [indomus/forum#2502] [HUB: Home Assistant]
Ciao a tutti, sto iniziando con le automazioni... in corridoio ho 4 interruttori wifi che accendono la stessa luce. Per farli funzionare ho creato 8 automazioni, 4 accendono e 4 spengono.
1 automazione:
Attivazione: interruttore 1 ON
Azioni: interruttori 2, 3, 4 ON
2 automazione:
Attivazione: interruttore 2 ON
Azioni: interruttori 1, 3, 4 ON
3 automazione:
Attivazione: interruttore 3 ON
Azioni: interruttori 1, 2, 4 ON
4 automazione:
Attivazione: interruttore 4 ON
Azioni: interruttori 1, 2, 3 ON
E altre 4 con OFF.
Funziona tutto ma vorrei sapere se esiste un modo più veloce e meno laborioso per fare ciò.
Grazie

**Comment by 50494554524F** [comment]
usa i gruppi... o uno switch derivato che combini lo stato dei quattro

**Reply by dockydock** [reply]
Grazie per l'aiuto, ho iniziato da 3 settimane e quello che hai scritto è  troppo... potresti fare un'esempio di codice?

**Reply by 50494554524F** [reply]
allora usa direttamente switch group

https://www.home-assistant.io/integrations/group/

combinando i 4 interruttori in un gruppo questi verranno visti come fosse uno solo, quindi tutto diventerà più semplice da gestire...

**Reply by dockydock** [reply]
Grazie mille
